packages/cli/src/commands/deploy/index.ts

```diff
@@ -355,6 +305,66 @@ export default async (client: Client): Promise<number> => {
     throw new Error(`"org" is not defined`);
   }
 
+  // build `--prebuilt`
+  if (argv['--prebuilt']) {
```

Comment:
I can't test this because I can't get `vc build` to work. When I try it in a project dir:

```
$  vc build
Vercel CLI 30.2.1-e945958
? Please select a Project: vercel-cli-monorepo-test-two
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
Error: ENOENT: no such file or directory, chdir '/Users/smassa/source/demo/vercel-cli-monorepo-test/packages/one' -> '/Users/smassa/source/demo/vercel-cli-monorepo-test/packages/one/packages/one'
```

and when I try it in the root, I hit an infinite loop:

```
$  vc build
Vercel CLI 30.2.1-e945958
? Please select a Project: vercel-cli-monorepo-test-one
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Please select a Project: vercel-cli-monorepo-test-one
> Downloading `preview` Environment Variables for Project vercel-cli-monorepo-test-one
✅  Created .vercel/.env.preview.local file [109ms]

> Downloading project settings
✅  Downloaded project settings to ~/source/demo/vercel-cli-monorepo-test/packages/one/packages/one/.vercel/project.json [0ms]
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Please select a Project: vercel-cli-monorepo-test-one
> Overwriting existing .vercel/.env.preview.local file
> Downloading `preview` Environment Variables for Project vercel-cli-monorepo-test-one
> No changes found.
✅  Updated .vercel/.env.preview.local file [113ms]

> Downloading project settings
✅  Downloaded project settings to ~/source/demo/vercel-cli-monorepo-test/packages/one/packages/one/.vercel/project.json [0ms]
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Please select a Project:
● vercel-cli-monorepo-test-one
○ vercel-cli-monorepo-test-two

...
```